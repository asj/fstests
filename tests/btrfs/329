#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright (c) 2025 Oracle.  All Rights Reserved.
#
# FS QA Test 329
#
# Verify sysfs knob input syntax for read_policy round-robin
#
. ./common/preamble
_begin_fstest auto quick

. ./common/sysfs
. ./common/filter

_require_test
_require_fs_sysfs_attr_policy $TEST_DEV read_policy round-robin

_verify_sysfs_syntax $TEST_DEV read_policy round-robin 4096

status=0
exit
