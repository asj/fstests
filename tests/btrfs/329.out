QA output created by 329
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
Invalid argument
